### DISCUSSION TITLE
Welcome to openparsec Discussions!

### DISCUSSION THREAD

**Discussion by uberlinuxguy** [OpenParsec/openparsec#56] [Announcements]
## 👋 Welcome Valiant Coder or Gamer looking for something new... prepare yourself ...
So I came back to this after being a few years off, and decided I would start some GitHub discussions

Here's what you can do!!!!

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

So let me start this discussion, is there still interest in OpenParsec?  Comment down below, use the emotes, whatever you feel like.

**Comment by Cheeseness** [comment]
Hi hi! My name is Cheese, and I'm an independent/freelance game dev based on the tiny island of Tasmania. I like to contribute to F/OSS projects from time to time, and I've been known to release some of my own projects under Free Software licences as well.

I have very fond memories of playing Parsec at LAN parties, and used to make sure I always had a couple of spare joysticks for anybody who didn't have one, but wanted to join in.

I said years ago that I'd like to do some work on Parsec's Starmap UI. So far, that hasn't happened (too many projects, too little time ;_; ), but it's still something I'd like to do if/when I find myself with the right combination of free time and motivation.

I'm super happy to offer support and encouragement for the community where I'm able. I'd love to see the project pick up little more momentum! :)
